aws/middleware/middleware.go

```diff
@@ -101,3 +101,23 @@ func GetResponseMetadata(metadata middleware.Metadata) (v ResponseMetadata) {
 func setResponseMetadata(metadata *middleware.Metadata, responseMetadata ResponseMetadata) {
 	metadata.Set(responseMetadataKey{}, responseMetadata)
 }
+
+// AddRequestInvocationIDMiddleware adds RequestInvocationIDMiddleware to the middleware stack
+func AddRequestInvocationIDMiddleware(stack *middleware.Stack, optFns ...func(*RequestInvocationIDMiddleware)) {
+	requestInvocationIDMiddleware := RequestInvocationIDMiddleware{}
+	for _, fn := range optFns {
+		fn(&requestInvocationIDMiddleware)
+	}
+
+	stack.Build.Add(requestInvocationIDMiddleware, middleware.After)
+}
+
+// AddAttemptClockSkewMiddleware adds AttemptClockSkewMiddleware to the middleware stack
+func AddAttemptClockSkewMiddleware(stack *middleware.Stack, optFns ...func(*AttemptClockSkewMiddleware)) {
+	attemptClockSkeyMiddleware := AttemptClockSkewMiddleware{}
+	for _, fn := range optFns {
+		fn(&attemptClockSkeyMiddleware)
+	}
+
+	stack.Deserialize.Add(attemptClockSkeyMiddleware, middleware.After)
```

Comment:
Do these helpers need functional options? It probably will be confusing to have these when there are no options that could be set.